contracts/UniswapV3Staker.sol

```diff
@@ -114,7 +114,7 @@ contract UniswapV3Staker is IUniswapV3Staker, Multicall {
 
         incentives[incentiveId].totalRewardUnclaimed += reward;
 
-        TransferHelper.safeTransferFrom(address(key.rewardToken), msg.sender, address(this), reward);
+        TransferHelperExtended.safeTransferFrom(address(key.rewardToken), msg.sender, address(this), reward);
```

Comment:
this is the only place it's strictly necessary, but adding it elsewhere doesn't hurt